modules/serverbidBidAdapter.js

```diff
@@ -79,114 +134,87 @@ export const spec = {
       if (placement.networkId && placement.siteId) {
         data.placements.push(placement);
       }
-    });
-
-    ret.data = JSON.stringify(data);
-    ret.bidRequest = validBidRequests;
-    ret.url = ENDPOINT_URL;
+    }
 
-    return ret;
-  },
+    if (data.placements.length) {
+      ajax(config.BASE_URI, _responseCallback, JSON.stringify(data), { method: 'POST', withCredentials: true, contentType: 'application/json' });
+    }
+  }
 
-  /**
-   * Unpack the response from the server into a list of bids.
-   *
-   * @param {*} serverResponse A successful response from the server.
-   * @return {Bid[]} An array of bids which were nested inside the server.
-   */
-  interpretResponse: function(serverResponse, bidRequest) {
+  function _responseCallback(result) {
```

Comment:
This doesn't seem to be the latest version of _responseCallback. Take a look at the one in the `serverbid-refresh-fix` branch.